pkg/informers/service/informer_service_test.go

```diff
@@ -0,0 +1,245 @@
+package service_test
+
+import (
+	"fmt"
+	"testing"
+
+	appservice "github.com/codeready-toolchain/registration-service/pkg/application/service"
+	"github.com/codeready-toolchain/registration-service/pkg/informers"
+	"github.com/codeready-toolchain/registration-service/pkg/informers/service"
+	"github.com/codeready-toolchain/registration-service/pkg/kubeclient"
+	"github.com/codeready-toolchain/registration-service/test"
+
+	toolchainv1alpha1 "github.com/codeready-toolchain/api/api/v1alpha1"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/tools/cache"
+)
+
+func TestRunInformerServiceSuite(t *testing.T) {
+	suite.Run(t, &TestInformerServiceSuite{test.UnitTestSuite{}})
+}
+
+type TestInformerServiceSuite struct {
+	test.UnitTestSuite
+}
+
+// Testing the Informer Service is mainly about ensuring the right types are being returned since informers use
+// dynamic clients and if a Lister is setup incorrectly it can lead to bugs where we try to convert the unstructured object
+// to the wrong type. This is possible and will not return any errors!
+func (s *TestInformerServiceSuite) TestInformerService() {
+
+	s.Run("masteruserrecords", func() {
+		murLister := fakeLister{
+			objs: map[string]*unstructured.Unstructured{
+				"johnMur": {
+					Object: map[string]interface{}{
+						"spec": map[string]interface{}{
+							"tierName": "deactivate30",
+							"userID":   "john-id",
+							"userAccounts": []map[string]interface{}{
+								{
+									"targetCluster": "member1",
+								},
+							},
+						},
+					},
+				},
+			},
+		}
+
```

Comment:
you could also add another MUR with a different name as an additional noise

Reply:
done in d28349e0f8b60e6f3ffef74e239af1f56df15a3e